ospf6d/ospf6_top.h

```diff
@@ -153,6 +153,10 @@ DECLARE_QOBJ_TYPE(ospf6);
 #define OSPF6_DISABLED    0x01
 #define OSPF6_STUB_ROUTER 0x02
 #define OSPF6_FLAG_ASBR   0x04
+#define OSPF6_MAX_IF_ADDRS 100
+#define OSPF6_MAX_IF_ADDRS_JUMBO 200
+#define OSPF6_DEFAULT_MTU 1500
+#define OSPF6_JUMBO_MTU 9000
```

Comment:
Neither 1500 nor 9000 are valid MTU numbers to hardcode here.  You can either assume 1280 == IPv6 minimum, or 64k (minus some bytes) for fragmentation.  For anything else, the only correct approach is to rely on _actual_ MTU reported by the interface.